src/System.Collections.Specialized/src/System/Collections/Specialized/OrderedDictionary.cs

```diff
@@ -253,8 +265,14 @@ public void Clear()
             {
                 throw new NotSupportedException(SR.OrderedDictionary_ReadOnly);
             }
-            objectsTable.Clear();
-            objectsArray.Clear();
+            if (_objectsTable != null)
```

Comment:
here we can just do `_objectsTable?.Clear()` ?

Reply:
I'd prefer use one style for whole file. Currently no '??' and '?' operators is in the file.

Reply:
Ok, makes sense to be consistent across the whole file.